util/dvsim/utils.py

```diff
@@ -343,3 +343,82 @@ def color_cell(cell, cclass, indicator="%"):
         for item in subst_list:
             text = text.replace(item, subst_list[item])
     return text
+
+
+def gen_css_style(use_css_style_for_web):
+    ## css style for web only
+    if use_css_style_for_web:
```

Comment:
I think you are right, i will add the `util/dvsim/style.css` back.
If we use the `web` style for email, the format sort of messed up. That is why I have two delete the following items:
```
  padding-left: 40px;\
  padding-right: 40px;\
  margin: 0 auto;\
  display: flex;\
  flex-direction: column;\
```
It will be simpler if that works for both. Let me try if the web version looks good without these settings.

![image](https://user-images.githubusercontent.com/11466553/80176042-3ea39b00-85ac-11ea-8f6b-0b53f0d599da.png)

Reply:
How about this updated code? We can still keep the `style.css` for web-server, but inline build-in `css_style` we only support the email format.

Reply:
Looks like the problem is with email clients misbehaving with embedded styles. See this:

Reply:
Sure, thanks for the link. Let me try it!